### PR TITLE
Block Web UI until service is ready

### PR DESCRIPTION
**Pull request by wbreza**
This PR addresses the issue where the UI allows you to attempt to create new items before the default list has loaded.  
This adds a `ready` state that will disable the UI input controls until the required service calls have completed successfully.

- [x] Disables input controls if list is not ready
- [x] Adds loading spinner animation while list items are loading
- [x] Adder shimmer loading effect to list title

### CODE REVIEW

File: templates/todo/web/react-fluent/src/components/todoItemListPane.tsx [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import { CommandBar, DetailsList, DetailsListLayoutMode, ICommandBarItemProps, IStackStyles, Selection, Stack, IIconProps, SearchBox, Text, IGroup, IColumn, MarqueeSelection, FontIcon, IObjectWithKey, CheckboxVisibility, IDetailsGroupRenderProps, getTheme } from '@fluentui/react';
+import { CommandBar, DetailsList, DetailsListLayoutMode, IStackStyles, Selection, Stack, IIconProps, SearchBox, Text, IGroup, IColumn, MarqueeSelection, FontIcon, IObjectWithKey, CheckboxVisibility, IDetailsGroupRenderProps, getTheme } from '@fluentui/react';
```
Comment (jongio) [inline templates/todo/web/react-fluent/src/components/todoItemListPane.tsx:1]: Can we have a "loading" message of some sorts so the user knows why the UI isn't active?

Comment (wbreza) [reply templates/todo/web/react-fluent/src/components/todoItemListPane.tsx:1]: Yes

**Comment by weikanglim** [comment]
Were you able to run the playwright tests to see if they still work with the change?

**Review by jongio — APPROVED** [review]

**Comment by wbreza** [comment]
> Were you able to run the playwright tests to see if they still work with the change?

Yes, playwright tests are passing as expected.

**Pull request merged**